.github/workflows/cron-protocol-test-with-coverage.yml

```diff
@@ -0,0 +1,31 @@
+# This tests are pretty slow and flacky
+name: Protocol tests with coverage
+# By default the sha where it runs is latest commit on default branch
+# https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule
+on:
+  schedule:
+  - cron: 0 13 * * *
+  workflow_dispatch:
+
+jobs:
+  # This seems to take ages and last codecov report is from 2021-02-25 :/
+  # https://app.codecov.io/gh/celo-org/celo-monorepo/commits
+  protocol-test-with-code-coverage:
+    runs-on: ["self-hosted", "monorepo"]
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install yarn dependencies
+        run: yarn install
+      - name: Build packages
+        run: yarn build --ignore docs --include-dependencies
+      - name: Run protocol tests with coverage
+        run: yarn --cwd packages/protocol test:coverage
+      - name: Store results
+        uses: actions/upload-artifact@v3
+        with:
+          name: protocol-coverage
+          path: packages/protocol/coverage
+      - name: Upload results to codecov
+        run: yarn codecov -F protocol
+        env:
+          CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
Protocol coverage tests fails because a timeout. I've tried to run during 10 hours but they did not conclude. As this is a problem present with independence of the GitHub migration, I'd say we can address in a future issue/PR.